opdracht 1.js

```diff
@@ -0,0 +1,30 @@
+import {inventory} from "./main.js";
+import {tvWeSell} from "./opdracht-4.js";
+
+const tvType = inventory.map((tvs) => {
+    return  tvs.type;
+})
+console.log(tvType);
+
+const soldOut = inventory.filter((tvs) => {
+    return tvs.originalStock - tvs.sold === 0;
+})
+console.log(soldOut)
+
+const ambilight = inventory.filter((tvs) => {
+    return tvs.options.ambiLight === true;
```

Comment:
`boolean_waarde === true` is altijd gelijk aan `boolean_waarde`, dus in dit geval
```suggestion
    return tvs.options.ambiLight;
```